src/Murder.Editor/Architect.cs

```diff
@@ -174,18 +174,18 @@ private void QuitToEditor()
         
         internal void PlayGame(bool quickplay, Guid? startingScene = null)
         {
-            startingScene ??= Profile.StartingScene;
+            Guid actualStartingScene = startingScene ?? Profile.StartingScene;
 
             // Data.ResetActiveSave();
 
-            if (!quickplay && startingScene == Guid.Empty)
+            WorldAsset? world = actualStartingScene != Guid.Empty ? Data.TryGetAsset<WorldAsset>(actualStartingScene) : null;
+            if (!quickplay && world is null)
             {
                 GameLogger.Error("Unable to start the game, please specify a valid starting scene on \"Game Profile\".");
                 return;
             }
 
-            if (Game.Data.TryGetAsset<WorldAsset>(startingScene.Value) is WorldAsset world && 
-                !world.HasSystems)
+            if (world is { HasSystems: false })
```

Comment:
I didn't know you could do that!!!

Reply:
Yeah, this checks for null and the prop you want in a quite nice way :)